external-docs/docs/quick-start/index.md

```diff
@@ -7,9 +7,9 @@ The suggested order for to navigate through the quick start pathway is as follow
 1. [Installation of csvcubed](./installation.md) - install python with csvcubed.
 2. [Designing a CSV](./designing-csv.md) - learn how to structure your data for csvcubed.
 3. [Building a CSV-W](./build.md) - build a CSV-W cube from a well structured CSV.
-4. [Describing your CSV](./describing-csv.md) - add metadata to your CSV-W describing your dataset.
-5. [Building 5* Linked Data](./linking-data.md) - use existing definitions to integrate into the linked data cloud.
+4. [Inspecting a CSV-W (Optional)](./inspect.md) - inspect the contents of a CSV-W to display and validate its metadata.
```

Comment:
The comment has disappeared but could you move this step to the end of the list?